### ISSUE TITLE
Could I use params within the get method ?

### ISSUE THREAD

**Issue by Tei1988** [nyamadori/juknife#4]
Hi. 

I want to scrape some same style pages, which have different urls.

I try to call the get method like below:
```ruby
get "http://#{params[:host]}"
```

But, NoMethodError is raised.
```ruby
irb(main):002:0> result = HogeFugaScraper.new.scrape(host: 'example.com')
NoMethodError: undefined method `[]' for nil:NilClass
	from /path/to/the/program/main.rb:22:in `block in <class:HogeFugaScraper>'
	from /path/to/.anyenv/envs/rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/juknife-0.1.1/lib/juknife/request/runner.rb:14:in `instance_eval'
	from /path/to/.anyenv/envs/rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/juknife-0.1.1/lib/juknife/request/runner.rb:14:in `initialize'
	from /path/to/.anyenv/envs/rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/juknife-0.1.1/lib/juknife/scraper.rb:8:in `new'
	from /path/to/.anyenv/envs/rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/juknife-0.1.1/lib/juknife/scraper.rb:8:in `initialize'
	from (irb):2:in `new'
	from (irb):2
	from /path/to/.anyenv/envs/rbenv/versions/2.4.0/bin/irb:11:in `<main>'
```

I want to use params within the get method.
Could I get some help ?

**Comment by nyamadori** [comment]
Thank you for trying this library!

If you use the parameters in the DSL, you need to give the URL with the block as follows:

``` ruby
get { "http://#{params[:host]}" }
```

Because params is not yet set when the Scraper constructor is invoked. By using blocks the scraper can evaluate the params when calling the `#scrape` method.

**Comment by nyamadori** [comment]
@Tei1988 
Other methods (e.g. `user_agent` `body`) also needs the block when you use `params` option in the DSL.

**Comment by nyamadori** [comment]
Sorry, the following code raises #8 errors in v0.1.2. So please update version to v0.1.3. This problem fixed at v0.1.3!

```
get { "http://#{params[:host]}" }
```
